platform-api/che-core-api-builder/src/main/java/org/eclipse/che/api/builder/internal/Builder.java

```diff
@@ -444,6 +444,7 @@ public Boolean call() throws Exception {
                 try {
                     ProcessBuilder processBuilder = new ProcessBuilder().command(commandLine.toShellCommand()).directory(
                             configuration.getWorkDir()).redirectErrorStream(true);
+                    modifyCommandEnvironment(processBuilder.environment(), configuration);
                     Process process = processBuilder.start();
```

Comment:
processBuilder.start() - need to be wrapped by some timer , so we can know the actual time spent in the command execution. - can be in another future change